qa/tasks/scripts/ses_install_rgw_part2.sh

```diff
@@ -3,10 +3,10 @@ set -ex
 declare -a storage_minions=("$@")
 
 random_minion_fqdn=${storage_minions[0]}
```

Comment:
there is no sense to use 'random_' prefix in this script, could you please get rid of it.
as I understand the script just takes a list of minions and does some similar operations.